client/state/plugins/wporg/actions.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:wporg-data:actions' );
+
+/**
+ * Internal dependencies
+ */
+import wporg from 'lib/wporg';
+import debounce from 'lodash/function/debounce';
+import utils from 'lib/plugins/utils';
+import { RECEIVE_WPORG_PLUGIN_DATA } from 'state/action-types';
+
+/**
+ * Constants
+ */
+const _LIST_DEFAULT_SIZE = 24,
+	_DEFAULT_FIRST_PAGE = 0;
+
+/**
+ *  Local variables;
+ */
+let _fetching = {},
+	_fetchingLists = {},
+	_currentSearchTerm = null,
+	_lastFetchedPagePerCategory = {},
+	_totalPagesPerCategory = {};
```

Comment:
These vars seem to be unused. Which is good, because the less stateful our modules are, the better.